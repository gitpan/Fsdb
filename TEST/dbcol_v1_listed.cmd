prog='dbcol'
args='mean'
cmp='diff -cb '
