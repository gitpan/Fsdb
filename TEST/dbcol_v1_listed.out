#fsdb -R C mean
mean: 37.25

mean: 271.2

#  | /home/johnh/BIN/DB/dbmultistats experiment duration
#  | /home/johnh/BIN/DB/dblistize 
#   | dbcol mean
